src/script/entity/Conversation.ts

```diff
@@ -964,12 +964,12 @@ export class Conversation {
   /**
    * Get the last delivered message.
    */
-  getLastDeliveredMessage(): Message | undefined {
+  private getLastDeliveredMessage(): Message | undefined {
     return this.messages()
       .slice()
       .reverse()
       .find(messageEntity => {
-        const isDelivered = messageEntity.status() >= StatusType.DELIVERED;
+        const isDelivered = [StatusType.DELIVERED, StatusType.SEEN].includes(messageEntity.status());
```

Comment:
This is the fix. 
For more explanation, here is what the enum looks like

```ts
export enum StatusType {
  DELIVERED = 3,
  FAILED = 0,
  FEDERATION_ERROR = 5,
  SEEN = 4,
  SENDING = 1,
  SENT = 2,
  UNSPECIFIED = -1,
}
```

as you can see `FEDERATION_ERROR` has a higher value than `DELIVERED` and thus caused the bug